tests/trainer/test_fsdp.py

```diff
@@ -13,10 +15,13 @@
 from tests.common import (EmbeddedWeightTiedModel, RandomClassificationDataset, SimpleModel, SimpleWeightTiedModel,
                           world_size)
 
+_INIT_DEVICES = ['cpu', 'meta', 'mixed']
```

Comment:
Do we also want to do cuda (and mark with gpu)?

Reply:
I can...do you think that is actually testing something new and interesting?

Reply:
not really...
just breadth testing 🤷‍♂️

Reply:
added